### PR TITLE
First pass at gpu stats for macs

### PR DESCRIPTION
**Pull request by vanpelt**

### CODE REVIEW

**Review by farizrahman4u — COMMENTED** [review]
Do we *really* need to check in C code in client repo? Can we download the binary during install?

**Comment by vanpelt** [comment]
> Do we _really_ need to check in C code in client repo? Can we download the binary during install?
> 
> Edit: NVM, just saw that you are bundling the binary in `vendor/` (I assumed user had to build following the readme). All good.

Yeah, right now I just build manually if I need to cur a new binary.  I have an apple developer account so we do code signing and all that fun stuff.  Figured we want to have the actual source code checked in and putting it here could help other users that want to access this information from Mac OS.

File: pyproject.toml [hunk 1]
```diff
@@ -13,6 +14,6 @@ exclude = '''
 | tests/notebooks/
 '''
 include = '''
-  wandb/
-| tests/
+  wandb/*\.pyi?$
+| tests/*\.pyi?$
```
Comment (raubitsj) [inline pyproject.toml:18]: This breaks black formatting/checking as it doesnt handle recursive dirs

**Review by raubitsj — CHANGES_REQUESTED** [review]

**Review by raubitsj — COMMENTED** [review]
otherwise, looks good

File: pyproject.toml (hunk 1 above)
Comment (raubitsj) [reply pyproject.toml]: fixed this in your branch

**Review by raubitsj — APPROVED** [review]

**Pull request merged**